adoc/retail_migr_chap_slepostosumaind.adoc

```diff
@@ -0,0 +1,10 @@
+[[retail_migr.sect.slepostosuma.s2a]]
+== Scenario 2a—Migrate SLEPOS ({susemgr} Installed Independantly)
```

Comment:
Remove Scenario 2a prefix.
`Migrate {slepos} to already existing independent {susemgr}`